functions.py

```diff
@@ -0,0 +1,32 @@
+import boto3
+import zipfile, os
+import glob
+client = boto3.client('cloudformation')
```

Comment:
Specify region wherever possible as a precautionary measure.
```suggestion
DEFAULT_REGION = "ap-south-1"
client = boto3.client('cloudformation', region_name=DEFAULT_REGION)
```

Reply:
Done